config/configgrpc/configgrpc.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package configgrpc defines the gRPC configuration settings.
+package configgrpc
+
+import "time"
+
+// GRPCSettings defines common settings for a gRPC configuration.
+type GRPCSettings struct {
+	// The headers associated with gRPC requests.
+	Headers map[string]string `mapstructure:"headers"`
+
+	// The target to which the exporter is going to send traces or metrics,
+	// using the gRPC protocol. The valid syntax is described at
+	// https://github.com/grpc/grpc/blob/master/doc/naming.md.
+	Endpoint string `mapstructure:"endpoint"`
+
+	// certificate file for TLS credentials of gRPC client. Should
+	// only be used if `secure` is set to true.
+	CertPemFile string `mapstructure:"cert-pem-file"`
+
+	// Whether to enable client transport security for the exporter's gRPC
+	// connection. See [grpc.WithInsecure()](https://godoc.org/google.golang.org/grpc#WithInsecure).
```

Comment:
nit: does markup link works for godoc?

Reply:
I copied those comments from the original opencensus exporter config, but [godoc documenting go code](https://blog.golang.org/godoc-documenting-go-code) says:

> URLs will be converted to HTML links; no special markup is necessary.

So I think we can remove the markups and leave only the links